base/data/Campaign.js

```diff
@@ -47,6 +48,56 @@ Campaign.fetchFor = (advert,status=KStatus.DRAFT) => {
 	return pvc;
 };
 
+/**
+ * Get the master campaign of a multi campaign object
+ * @param {Advertiser|Agency} multiCampaign 
+ * @returns {Campaign}
+ */
+Campaign.fetchMasterCampaign = (multiCampaign, status=KStatus.DRAFT) => {
+    let pvCampaign = getDataItem({type:C.TYPES.Campaign,status,id:multiCampaign.campaign});
+    return pvCampaign && pvCampaign.value;
+}
+
+/**
+ * Get all campaigns matchin an advertiser
+ * @param {Advertiser} vertiser
+ * @returns {Campaign[]}
+ */
+Campaign.fetchForAdvertiser = (vertiser, status=KStatus.DRAFT) => {
```

Comment:
Note that this isn't how Impact Hub does it -- which could lead to a mismatch. I think ImpactHub fetches a list of Adverts, then gets Campaign ids from the adverts.